octobot_trading/exchange_data/contracts/future_contract.pxd

```diff
@@ -24,6 +24,8 @@ cdef class FutureContract(margin_contract.MarginContract):
 
     cdef readonly double minimum_tick_size
 
+    cdef public object take_profit_stop_loss_mode
```

Comment:
Why not readonly + setter like other attributes?

Reply:
I don't really see the benefit of adding a setter here, what do you want to put in it ?